src/components/services/ContactsLoader.ts

```diff
@@ -304,6 +304,18 @@ const merchantPartners = [
     jobTitle: '',
     familyName: '',
     recordID: ''
+  },
+  {
+    givenName: 'Rango Exchange',
+    hasThumbnail: true,
+    thumbnailPath: 'rango.png',
```

Comment:
Please submit this png to us as part of the PR in the PR comments. It should be square 256x256 or 512x512 png with transparent background

Reply:
Here is the icon:
https://rango.exchange/logo512.png